Tests/Fixtures/Migrations/Migration001.php

```diff
@@ -6,26 +6,11 @@
 
 use Doctrine\DBAL\Schema\Schema;
 use Doctrine\Migrations\AbstractMigration;
-use Symfony\Component\DependencyInjection\ContainerAwareInterface;
-use Symfony\Component\DependencyInjection\ContainerInterface;
 
-class Migration001 extends AbstractMigration implements ContainerAwareInterface
```

Comment:
The `ContainerAwareInterface` is going away in Symfony 7. My PR simply makes the tests pass, but we probably have to think about a proper replacement for this feature.